Example/PaymentSheet Example/PaymentSheetUITest/EmbeddedUITest.swift

```diff
@@ -28,4 +28,177 @@ class EmbeddedUITests: PaymentSheetUITestCase {
         // ...should cause Alipay to no longer be the selected payment method, since it is not valid for setup.
         XCTAssertFalse(app.staticTexts["Payment method"].exists)
     }
+
+    func testSingleCardCBC_update_and_remove() {
+        var settings = PaymentSheetTestPlaygroundSettings.defaultValues()
+        settings.mode = .paymentWithSetup
+        settings.uiStyle = .paymentSheet
+        settings.customerKeyType = .legacy
+        settings.customerMode = .new
+        settings.merchantCountryCode = .FR
+        settings.currency = .eur
+        settings.applePayEnabled = .on
+        settings.apmsEnabled = .off
+
+        loadPlayground(app, settings)
+
+        app.buttons["Present PaymentSheet"].waitForExistenceAndTap()
+
+        try! fillCardData(app, cardNumber: "4000002500001001", postalEnabled: true)
+
+        // Complete payment
+        app.buttons["Pay €50.99"].tap()
+        XCTAssertTrue(app.staticTexts["Success!"].waitForExistence(timeout: 10))
+
+        // Switch to embedded mode kicks off a reload
+        app.buttons["embedded"].waitForExistenceAndTap(timeout: 5)
+        app.buttons["Present embedded payment element"].waitForExistenceAndTap()
+
+        // Ensure card preference is cartes bancaires
+        XCTAssertTrue(app.buttons["•••• 1001"].waitForExistence(timeout: 3))
+        XCTAssertTrue(app.images["stp_card_cartes_bancaires"].waitForExistence(timeout: 3))
+        XCTAssertFalse(app.images["stp_card_visa"].waitForExistence(timeout: 3))
+
+        app.buttons["Edit"].waitForExistenceAndTap()
+        app.otherElements["Card Brand Dropdown"].waitForExistenceAndTap()
+        app.pickerWheels.firstMatch.swipeUp()
+        app.buttons["Done"].waitForExistenceAndTap()
+        app.buttons["Update"].waitForExistenceAndTap()
+        XCTAssertFalse(app.staticTexts["Update card brand"].waitForExistence(timeout: 3))
+
+        // Ensure card preference is switched to visa
+        XCTAssertTrue(app.buttons["•••• 1001"].waitForExistence(timeout: 3))
+        XCTAssertTrue(app.images["stp_card_visa"].waitForExistence(timeout: 3))
+        XCTAssertFalse(app.images["stp_card_cartes_bancaires"].waitForExistence(timeout: 3))
```

Comment:
We should hit done/back and make sure that the embedded view correctly updates. I'm not seeing this behavior in the branch.

Reply:
I'm not following your comment.

In this test, the UpdateCardViewController is presented, and CBC is updated. On line 66, update is tapped, and the sheet dismisses.  What behavior are you not seeing in the branch?